src/store/transaction.ts

```diff
@@ -32,7 +32,7 @@ export const showTxDetails = atom<boolean>({
   default: true,
 });
 
-export const requestTokenStore = atom<string>({
+export const requestTokenState = atom<string>({
```

Comment:
What was the rational for this change? we have a convention to use `Store` as the suffix for atoms. I'd be open to changing it to `State`, but this seems unrelated to the goal of this PR.

Reply:
If you could undo that change, the changes would be more clearly related to the cancel functionality.

Reply:
Yep, I can, I honestly just thought the convention of `Store` was a bit strange. Any documentation I've seen on Recoil uses `State` and I did see it in other places so I thought maybe switching would be preferred. I can change it back, but I lean towards liking `State` better.

Reply:
I agree, and we should! but for this PR the name change only pollutes the diff and makes it harder to track the changes for cancel. I think we have an opportunity to come up with whatever conventions we want, and i'm excited to find something that we're all happy with!

Reply:
Thanks for letting me know I should do that in a separate PR.